src/main/webapp/app/entities/course/course.model.ts

```diff
@@ -13,6 +21,7 @@ export class Course implements BaseEntity {
     public endDate: Moment;
     public onlineCourse = false; // default value
     public exercises: Exercise[];
+    public exerciseType: ExerciseType;
```

Comment:
What is this property? Can this (and the enum above) be deleted?

Reply:
I agree, this makes no sense here and is already defined in the exercise itself. Please delete!